composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```

Comment:
Might as well bump to 7.4 minimum

```suggestion
        "php": "^7.4 || ^8.0",
```

Reply:
good hint! requirement bumped to 7.4 minimum and the composer.lock is also updated

Reply:
:+1: